pyrevolve/genotype/neat_brain_genome/crossover.py

```diff
@@ -1,24 +1,32 @@
+import random
+
 class NEATCrossoverConf:
     def __init__(self):
         self.mate_average = True
         self.interspecies_crossover = True
 
 
-def standard_crossover(parent_individuals, genotype_conf, crossover_conf):
+def standard_crossover(parents, NeatCrossoverConf, crossover_conf, lsystem_conf):
     """
     Creates an child (individual) through crossover with two parents
 
-    :param parent_individuals: parent individuals to be used for crossover
-    :param genotype_conf: NEAT genotype configuration object
-    :param crossover_conf: NEAT genotype crossover configuration object
+    :param parents: parent individuals to be used for crossover
+    :param NeatCrossoverConf: NEAT genotype configuration object
+    :param crossover_conf: CrossoverConfig for lsystem
     :return: genotype result of the crossover
     """
-    assert len(parent_individuals) == 2
-    mother = parent_individuals[0].genotype
-    father = parent_individuals[1].genotype
-    child_genotype = mother.Mate(father,
-                                 crossover_conf.mate_average,
-                                 crossover_conf.interspecies_crossover,
-                                 genotype_conf.rng,
-                                 genotype_conf.multineat_params)
-    return child_genotype
+    assert len(parents) == 2
+
+    crossover_attempt = random.uniform(0.0, 1.0)
+    if crossover_attempt > crossover_conf.crossover_prob:
+         new_genotype = parents[0].genotype._brain_genome
+    else:
+        mother = parents[0]
```

Comment:
aren't parents of type individual? Should it not be `parents[0].genotype`?

Reply:
No parents here contain only the brain part of the genome